Docs/Getting Started Guide.md

```diff
@@ -6,6 +6,60 @@ ReSwift provides the infrastructure for `Store`s, `Action`s and `Reducer`s to en
 
 The following steps will describe how to set up the individual components for your ReSwift app.
 
+# General Setup and High Level Overview
+
```

Comment:
If we want to keep this section here, I would add a brief intro so that folks understands what's coming up. Something like:

"This section will show you how to set up the core ReSwift components in your app. The following sections will describe the individual components in depth."